rt_rubocop_defaults.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 0.77"
+  spec.add_dependency "rubocop", "~> 0.81"
```

Comment:
Current version is `0.82`, but it dropped the support for ruby `2.3`.

Reply:
i guess we could still update to 82, ruby 2.3 is not supported anymore anyways. old projects can stick to the old rt_rubocop_defaults i guess?

Reply:
well the defaults are forward compatible aren't the?

Reply:
I'll suggest doing the upgrade (and the breaking change) to `0.82` in a new PR.
With that we'll also can remove some renamed cops etc.